content/en/docs/instrumentation/erlang/resources.md

```diff
@@ -88,7 +91,7 @@ flattened and combined with `.`, so
 ## Custom resource detectors
 
 Custom resource detectors can be created by implementing the
-[`otel_resource_detector` behaviour](https://hexdocs.pm/opentelemetry/1.3.0/otel_resource_detector.html#callbacks)
+[`otel_resource_detector behaviour`](https://hexdocs.pm/opentelemetry/1.3.0/otel_resource_detector.html#callbacks)
```

Comment:
```suggestion
[`otel_resource_detector` behaviour](https://hexdocs.pm/opentelemetry/1.3.0/otel_resource_detector.html#callbacks)
```

Reply:
I put the behaviour into the backticks on purpose, but now looking at it, it makes no sense... let's stick with the current state and the comment, this is erlang specific lingua so we (docs team) should follow